tests/testthat/test-download_basemap.R

```diff
@@ -1,3 +1,16 @@
 test_that("download_basemap() returns error on incorrect parameters", {
-
+  ppi <- project_as_ppi(example_scan)
+  expect_error(
+    download_basemap("a"),
+    regexp = 'inherits(x, "ppi") is not TRUE',
+    fixed = TRUE
+  )
+  # CRS absent in
+  ppi_no_crs <- ppi
+  raster::crs(ppi_no_crs) <- NA
+  expect_error(
+    download_basemap(ppi_no_crs),
+    regexp = "Not a projected ppi, download_basemap() expects a ppi generated by project_as_ppi() with argument project=TRUE",
+    fixed = TRUE
+  )
```

Comment:
I would have expected `download_basemap(project_as_ppi(example_scan, project = FALSE))` to also return this error, but it doesn't!

Reply:
fixed a typo in my test, so while github will flag this as outdated but I still suspect there might be a bug here in `download_basemap()`